sp/src/game/server/physics_prop_ragdoll.cpp

```diff
@@ -832,7 +838,11 @@ void CRagdollProp::HandleFirstCollisionInteractions( int index, gamevcollisionev
 		info.SetDamage( m_iHealth );
 		info.SetAttacker( this );
 		info.SetInflictor( this );
+#ifdef EZ2
+		info.SetDamageType( DMG_CRUSH | DMG_ALWAYSGIB );
```

Comment:
Won't this change affect any ragdoll having a first collision interaction? Are there any possible effects this could have outside of the intended case? I can't think of anything bad that could happen, but it just feels odd to me to make a global change to all ragdolls just for stukabats.

Reply:
Unfortunately it's been so long that I don't remember the rationale behind this change, although I'm also struggling to think of any case outside of stukabats where ragdolls break on first impact.

Unless I'm missing something, the ragdoll break interaction seems to be based on functionality from stock Source, but it's possible that it was unused and adding `DMG_CRUSH | DMG_ALWAYSGIB` was needed to hook into E:Z2's ragdoll gib functionality.

Reply:
I think you're right. I looked through again and I see how ragdolls ignore damage that doesn't have the "DMG_ALWAYSGIB" type. So I think it's making sure ragdolls can respect break on touch